tests/general/LicenseCheckBearTest.py

```diff
@@ -0,0 +1,48 @@
+import os
+from queue import Queue
+
+from bears.general.LicenseCheckBear import LicenseCheckBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'licensecheck_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as fl:
+        contents = fl.read()
+
+    return contents
+
+
+@generate_skip_decorator(LicenseCheckBear)
+class LicenseCheckBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = LicenseCheckBear(Section('name'), Queue())
+
+    def test_license(self):
+        test_file_path = get_testfile_path('mit_license.py')
+        file_contents = [load_testfile(test_file_path)]
```

Comment:
Also `check_results` expects a list of lines, not a list of file-strings with the full content :) I think you should improve your `load_testfile` with

```
return file.splitlines(True)
```
Then you also don't need the square brackets here.